universal/CMakeLists.txt

```diff
@@ -78,9 +84,15 @@ endif()
 add_library(userver-internal-compile-options INTERFACE)
 userver_target_require_dwcas(userver-internal-compile-options INTERFACE)
 target_compile_features(userver-internal-compile-options INTERFACE cxx_std_17)
-target_compile_options(userver-internal-compile-options INTERFACE
-  "-Wall" "-Wextra" "-Wpedantic"
-)
+if (MSVC)
```

Comment:
Please no MSVC for now, Clang on Windows is enough :)
We are not ready for MSVC's bugs and lack of certain intrinsics